### DISCUSSION TITLE
winget setting.installBehavior.portablePackageUserRoot，Can it be set in a non-user directory?

### DISCUSSION THREAD

**Discussion by dong-lufei** [microsoft/winget-cli#2640] [General] [answered]
"installBehavior": {
        //"portablePackageUserRoot": "C:/Users/FooBar/Packages"
        "portablePackageUserRoot": "D:/winget-package"
}

**Comment by denelon** [comment]
@ryfu-msft can you handle this one?

**Comment by ryfu-msft** [comment, marked as answer]
@git136975643, yes this still works. I also tried it out and didn't encounter any issues.

**Comment by dong-lufei** [comment]
"installBehavior": {
      "defaultInstallRoot": "D:/winget-package"
  },

上面和这个的区别？

**Comment by dong-lufei** [comment]
"portablePackageMachineRoot": "D:/winget-package"

The difference between these settings？
